js/index.js

```diff
@@ -127,17 +135,18 @@ var Controller = {
 
   handleDrop: function(event) {
     var column = $(event.target).closest('.column');
-    if (column.length() > 0) {
+    if (column.length > 0) {
       var id = event.dataTransfer.getData('text');
       Model.moveTask(id, column.attr('id'));
       View.renderBoard();
     }
   },
 
   handleLoad: function() {
+    console.log("ajax request");
     $.ajax({
       type: 'GET',
-      url: 'http:/jacobfriedmann.com:3000/todos?num=1',
+      url: 'jacobfriedmann.com:3000/todos?num=1',
       success: function(data) {
         data.tasks.forEach(function(task) {
```

Comment:
There is no `tasks` key within the `data` returned. Instead, you just want to loop through `data` itself.